packages/client/src/components/Visual.test.tsx

```diff
@@ -31,4 +31,38 @@ describe("Visual", () => {
     expect(screen.queryByText("title")).toBeInTheDocument();
     expect(screen.queryByTestId(FIGCAPTION)).toBeInTheDocument();
   });
+
+  it("renders correctly with on error", () => {
+    const { container } = render(
+      <Visual alt="alt text" src="folder/image.jpg" title="title" />
+    );
+
+    expect(screen.queryByAltText("alt text")).toBeInTheDocument();
+    expect(screen.queryByText("title")).toBeInTheDocument();
+    expect(screen.queryByTestId(FIGCAPTION)).toBeInTheDocument();
+
+    let img = container.querySelector("img");
+    expect(img).not.toBeNull();
+    if (img) {
+      fireEvent(img, new Event("error"));
+      expect(img).toHaveStyleRule("border", "1px solid #767676");
+    }
+  });
+
+  it("renders correctly with on load", () => {
```

Comment:
This `load` test is a duplicate of the `error` and you test the same StyleRule (which only appears on `loaded`, this `errored` is also `loaded`). Can you replace that with a corresponding StyleRule test like:

error: `expect(img).toHaveStyleRule("padding", "33px");`
load: `expect(img).not.toHaveStyleRule("padding", "33px");`

Reply:
ping @svenjens

Reply:
I fixed this